src/KW_label_internal.c

```diff
@@ -70,7 +70,7 @@ void PaintLabel(KW_Widget * widget, const KW_Rect * absolute, void * data) {
   };
 
   /* apply horizontal offset and icon offset */
-  dst.x += label->hoffset;
+  dst.x += label->hoffset+label->iconclip.w/2;
```

Comment:
Seems clang-format has not been executed in this one, if I'm not mistaken there should be spaces around `+` and `/` right?

Reply:
i may have forgot to run it on the file, thought i had. odd